.npmrc

```
@@ -1 +1,3 @@
 save-exact=true
+node-version=16.13.0
+engine-strict=true
```

```suggestion
engine-strict=true
node-version=16.13.0
npm-version=8.1.0
```

Refs: https://docs.npmjs.com/cli/v8/using-npm/config#engine-strict
Refs: https://docs.npmjs.com/cli/v8/using-npm/config#node-version
Refs: https://docs.npmjs.com/cli/v8/using-npm/config#npm-version